### ISSUE TITLE
Error installing 0.8.0

### ISSUE THREAD

**Issue by max-maurice** [simontaite/jquery.dirty#39]
Using `npm i jquery.dirty` brings up an error:

```
> jquery.dirty@0.8.0 preinstall [snip]\node_modules\jquery.dirty
> npx npm-force-resolutions

npx: installed 5 in 1.727s
ENOENT: no such file or directory, open './package-lock.json'
```

It seems to be caused by b54f03b1fd471199484929939a8244666de36898 adding 
```json
"preinstall": "npx npm-force-resolutions",
```

Installing 0.7.2 works fine.

References: 
https://github.com/oktadeveloper/schematics/issues/192#issuecomment-688720326

**Comment by EPGDigital** [comment]
package-lock.json is npm too, if you are using yarn the file wouldn't exist.

I reverted back to the 0.7.3 release

**Comment by max-maurice** [comment]
package-lock.json does exist. If I try to install 0.7.2 everything works fine. However, I have to upgrade to 0.8.0 because of #22

**Comment by simontaite** [comment]
Sorry about that, v0.8.1 now published to npm, should resolve this

**Comment by max-maurice** [comment]
Fixed with 0.8.1, thanks Simon! :-)
